mlir/lib/Target/SPIRV/SPIRVBinaryUtils.cpp

```diff
@@ -42,7 +42,7 @@ void spirv::appendModuleHeader(SmallVectorImpl<uint32_t> &header,
   // +-------------------------------------------------------------------------+
   // | Magic number                                                            |
   // +-------------------------------------------------------------------------+
-  // | Version number (bytes: 0 | major number | minor number | 0)             |
+  // | Version number (bytes: 0 | 0 | major number | minor number)             |
```

Comment:
This seems to contradict the text of the spec: https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#_physical_layout_of_a_spir_v_module_and_instruction (see section 2.3 "Physical Layout of a SPIR-V Module and Instruction")

Can you demonstrate that it's correct? For example, does `spirv-dis` agree with it?